gdcdatamodel/gdc_postgres_admin.py

```diff
@@ -434,9 +442,13 @@ def get_parser():
 def main(args=None):
     args = args or get_parser().parse_args()
 
+    # fix empty string passed as namespace
+    args.namespace = args.namespace or None
```

Comment:
can do something like `subparser.add_argument("-N", "--namespace", type=lambda x: None if not x else x)` I think.  or define a callable or something. but not really important